src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Navbar, Nav, Button, Form } from "react-bootstrap";
+import logo from "../../images/logo.png";
+import "./NavBar.css";
+
+function NavBar() {
+  return (
+    <Navbar variant="light" expand="lg" className="bgLight">
+      <Navbar.Brand href="/" exact="true">
+        <img src={logo} height="50" alt="Furry Friend" className="logoHeader" />
+      </Navbar.Brand>
+      <Navbar.Toggle aria-controls="basic-navbar-nav" />
+      <Navbar.Collapse id="basic-navbar-nav">
+        <Nav className="mr-auto">
+          <Nav.Link href="/about">About</Nav.Link>
```

Comment:
for all the Nav.Link please use the NavLink component from React router, using this link will cause a page refresh and won't be as smooth as react router